### PR TITLE
Improve Keystone Application Credential docs

### PR DESCRIPTION
**Pull request by zioproto**
**What this PR does / why we need it**:

It adds documentation for the changes pushed in #282

### CODE REVIEW

File: docs/using-client-keystone-auth.md [hunk 1]
```diff
@@ -108,6 +108,11 @@ arguments
 (`--keystone-url`, `--domain-name`, `--user-name`, `--password` and `--project-name`), respectively.
 If they are not specified, the user will be prompted to enter them at the time of the interactive
 session.
+It is also possible to authenticate with [Keystone Application Credentials](https://docs.openstack.org/keystone/queens/user/application_credentials.html).
```
Comment (jichenjc) [inline docs/using-client-keystone-auth.md]: suggest to use https://docs.openstack.org/keystone/latest/user/application_credentials.html
so we always recommend latest

Comment (zioproto) [reply docs/using-client-keystone-auth.md]: You are right ! Fixed

**Pull request merged**